src/example.ts

```diff
@@ -89,23 +84,26 @@ const samlConfig: SamlConfig = {
   RACComparison: "minimum",
   acceptedClockSkewMs: 0,
   attributeConsumingServiceIndex: "0",
-  authnContext: "https://www.spid.gov.it/SpidL1",
-  callbackUrl: "http://localhost:3000" + appConfig.assertionConsumerServicePath,
+  authnContext: process.env.AUTH_N_CONTEXT,
+  callbackUrl: `${process.env.ORG_URL}${process.env.ENDPOINT_ACS}`,
   // decryptionPvk: fs.readFileSync("./certs/key.pem", "utf-8"),
   identifierFormat: "urn:oasis:names:tc:SAML:2.0:nameid-format:transient",
-  issuer: "https://spid.agid.gov.it/cd",
-  logoutCallbackUrl: "http://localhost:3000/slo",
-  privateCert: fs.readFileSync("./certs/key.pem", "utf-8"),
+  issuer: process.env.ORG_ISSUER,
+  // logoutCallbackUrl: conf.organization.url + "/slo",
```

Comment:
Is this on purpose?

Reply:
This is a suggestion that was useful for me: it is needed to cover cases when the host that will receive the SAML responses is different from the URL used for the Issuer.

Reply:
@mastrogiovanni can you also uncomment this line please?